removefiles/activity_test.go

```diff
@@ -0,0 +1,89 @@
+package removefiles_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/go-logr/logr"
+	temporalsdk_activity "go.temporal.io/sdk/activity"
+	temporalsdk_testsuite "go.temporal.io/sdk/testsuite"
+	"gotest.tools/v3/assert"
+	tfs "gotest.tools/v3/fs"
+
+	"github.com/artefactual-sdps/temporal-activities/removefiles"
+)
+
+func TestActivity(t *testing.T) {
+	t.Parallel()
+
+	type args struct {
+		path   func(td *tfs.Dir) string
+		remove string
+	}
```

Comment:
We not reusing `removefiles.ActivityParams` instead of using a function for the path? You could create the test directory in the test arguments and return the path at once:

```go
params: &removefiles.ActivityParams{
    Path: tfs.NewDir(t, "remove-files-activity-test",
        tfs.WithFile(".DS_Store", "delete me!"),
        tfs.WithFile("keepme", "keep me!"),
    ).Path(),
    RemoveNames: "Thumbs.db, .DS_Store",
},
```

That way you could pass different directories to each scenario (not that is needed here).

Reply:
Because I didn't think of it. :laughing:  Nice solution! :D